pkg/bloomgateway/bloomgateway_test.go

```diff
@@ -77,7 +77,8 @@ func setupBloomStore(t *testing.T) *bloomshipper.BloomStore {
 		},
 	}
 
-	store, err := bloomshipper.NewBloomStore(schemaCfg.Configs, storageCfg, cm, nil, nil, logger)
+	reg := prometheus.NewPedanticRegistry()
```

Comment:
Why Pedantic registry instead of the usual default registry?

Reply:
There's no reason for using the PedanticRegistry, which, according to docs, is useful for testing implementation of collectors and/or metrics, but we don't do that here.